packages/office-ui-fabric-react/src/components/Persona/__snapshots__/Persona.test.tsx.snap

```diff
@@ -15,15 +15,6 @@ exports[`Persona renders Persona correctly with image 1`] = `
       style={undefined}
     >
       <div
-        aria-hidden="true"
-        className="ms-Persona-initials ms-Persona-initials--red undefined"
-        style={undefined}
-      >
-        <span>
-          KL
-        </span>
-      </div>
-      <div
```

Comment:
fixed this in here, too: https://github.com/OfficeDev/office-ui-fabric-react/pull/3171
to do it, I had to remove an empty line from the Persona test file 'styles' const before running `npm run update-snapshots`. did you make that change somewhere too?

Reply:
@lynamemi  Hm was that line space causing issues for you? I only ran `update-snapshots` but didn't make any changes to the test file itself. 

A previous PR of mine did make changes to Persona, but when it was merged, the snapshots got out of sync with master, so this PR is just to update the snapshot.

Reply:
Ahhh maybe I made a false connection then. It worked so I assumed that was why! I'll go ahead and remove the changes from my PR so it's less confusing. I'm assuming this one will get merged in before mine.